src/DeviceInterface/AppleDeviceInterface.js

```diff
@@ -174,23 +174,6 @@ class AppleDeviceInterface extends InterfacePrototype {
      * PM endpoints
      */
 
-    /**
-     * Sends credentials to the native layer
-     * @param {{username: string, password: string}} credentials
-     */
-    storeCredentials (credentials) {
-        return this.deviceApi.notify(createNotification('pmHandlerStoreCredentials', credentials))
-    }
```

Comment:
This was never used as we were using the generic `storeFormData`. It was dead code.